shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -48,6 +48,10 @@
 // The parent view handles clipping to its subviews.
 @interface ChildClippingView : UIView
 
+// Adds a blur filter to its layers.
+- (void)applyBackdropFilterWithRadius:
```

Comment:
It might be better to let this method take DLImageFilter as a parameter. Then inside this method, we can get information from the `DLImageFilter` object